tests/virt_autotest/virt_autotest_base.pm

```diff
@@ -28,6 +28,17 @@ sub get_script_run {
     die "You need to overload this func in your class";
 }
 
+sub svirt_cmd {
+    my ($self, $cmd, $args) = @_;
+    die 'Command not provided' unless $cmd;
+
+    $args->{ignore_return_code} ||= 0;
+    my $ret = console('svirt')->run_cmd($cmd);
+    diag "Command on S390X LPAR returned: $ret";
+    die 'Command on S390X LPAR failed' unless ($args->{ignore_return_code} || !$ret);
+    return $ret;
+}
+
```

Comment:
Sorry for giving the wrong suggestion to add this function in base class. It is not common function for all arches. Do you mind moving it to virt_autotest/virt_utils.pm?